docs/core/diagnostics/sample-debug-target.md

```diff
@@ -1,25 +1,25 @@
 ---
 title: Sample debug target
-description: Introduction to the sample debug target used in the scenarios.
+description: Introduction to the sample debug target used in the diagnostic tutorials.
 author: sdmaclea
 ms.author: stmaclea
-ms.date: 10/14/2019
+ms.date: 12/17/2019
 ---
 # Sample debug target
 
-The scenarios are implemented using a `webapi` target with methods that trigger undesirable behaviors for us to diagnose.
+The sample debug target is a simple `webapi` application. The sample triggers undesirable behaviors for the [.NET Core diagnostics tutorials](index.md#net-core-diagnostics-tutorials) to diagnose.
 
 ## Sample source
 
 The sample code is in [DiagnosticScenarios](https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios). The sample is a basic `webapi` with a single added file [DiagnosticScenarios.cs](https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios/Controllers/DiagnosticScenarios.cs).
```

Comment:
I think a link to the samples browser would be better than <https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios>

Reply:
I assume you are referring to https://github.com/dotnet/samples/blob/master/core/diagnostics/DiagnosticScenarios/Controllers/DiagnosticScenarios.cs